docs/Translation.md

```diff
@@ -323,7 +323,7 @@ If you need to translate messages in your own components, React-admin provides a
 ```jsx
 // in src/MyHelloButton.js
 import React from 'react';
-import { useTranslate } from 'react-admin';
+import { useTranslate } from 'ra-core';
```

Comment:
Nope, it was necessary in `ResettableTextField` because `ra-ui-materialui` does not depend on `react-admin`. 
`useTranslate` is re-exported from `react-admin` though.
Please revert